rootfs/etc/nginx/lua/balancer.lua

```diff
@@ -155,10 +168,12 @@ function _M.call()
   local ok
   ok, err = ngx_balancer.set_current_peer(host, port)
   if ok then
-    ngx.log(
-      ngx.INFO,
-      "current peer is set to " .. host .. ":" .. port .. " using lb_alg " .. tostring(get_current_lb_alg())
-    )
+    local algorithm = get_current_lb_alg()
+    if sticky.is_sticky(get_current_backend()) then
+      algorithm = "round robin with session affinity"
+    end
+    local message = string.format("current peer is set to %s:%s using lb_alg %s", host, port, algorithm)
+    ngx.log(ngx.INFO,  message)
```

Comment:
What value are we getting out of this special logging for the cost we are paying by calling `is_sticky` again?
I know it is not much cost to call that function but we can verify that the balancer is using session affinity by sending multiple requests and making sure the same upstream is being picked up just by looking at normal Nginx logs.

Reply:
In that case, should all logs be removed here?

Reply:
I'd just revert this particular change